scripts/docker/run.sh

```diff
@@ -75,31 +77,38 @@ eval set -- "$opts"
 os=`cat /etc/os-release | grep -oP "(?<=VERSION_CODENAME=).*"`
 sim_args="dds:=false robot:=sim_pub"
 display="true"
+gpu="true"
 mount="false"
 tagrepo="astrobee"
 dry_run="false"
+image=""
 
 # extract variables from options
 while [ "$1" != "" ]; do
     case $1 in
-        --help )                   usage
+        -h | --help )                   usage
```

Comment:
I think use of `getopt` earlier should have normalized `-h` to become `--help`, so this change isn't necessary. (It should be easy to check.) Same comment applies to the other arguments below.

Reply:
I did check , but it doesn't normalize it, it only checks if it's on the allowed list. I was also very confused by this...

Reply:
Bummer. Helps to know for future scripts, though!